index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/16d2250538.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Apple</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <a href="#"><div id="applebutton"><i class="fab fa-apple"></i></div></a>
+        <a href="#"><div id="storebutton">Store</div></a>
+        <a href="#"><div id="macbutton">Mac</div></a>
+        <a href="#"><div id="ipodbutton">iPod</div></a>
+        <a href="#"><div id="iphonebutton">iPhone</div></a>
+        <a href="#"><div id="ipadbutton">iPad</div></a>
+        <a href="#"><div id="itunesbutton">iTunes</div></a>
+        <a href="#"><div id="supportbutton">Support</div></a>
```

Comment:
- [ ] These ids are not being used. Kindly delete them. Often, prefer use classes over ids.